components/Stars.tsx

```diff
@@ -52,9 +52,9 @@ const Stars = ({
 
   return (
     <>
-      {title && <h3>{title}</h3>}
+      {title && <h3 className="center">{title}</h3>}
       <fieldset
-        className={styles.container}
+        className={`${styles.container} ${starStyles.container}`}
```

Comment:
Clever, but why is the star container different?

Reply:
It was because they used to share `display` styles! No longer relevant though.